src/AcoustID.Core/AcoustID.Core.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
```

Comment:
Please add a property group containing the assembly information:
```xml
  <PropertyGroup>
    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
    <GenerateDocumentationFile>true</GenerateDocumentationFile>
    <PackageId>AcoustID.NET</PackageId>
    <Description>AcoustID fingerprinter and webservice access.</Description>
    <Product>AcoustID</Product>
    <Copyright>Copyright © Christian Woltering 2016-2019</Copyright>
    <Authors>Christian Woltering</Authors>
    <PackageProjectUrl>https://github.com/wo80/AcoustID.NET</PackageProjectUrl>
    <RepositoryUrl>https://github.com/wo80/AcoustID.NET</RepositoryUrl>
    <AssemblyVersion>1.3.3.0</AssemblyVersion>
    <FileVersion>1.3.3.0</FileVersion>
    <PackageTags>music audio fingerprint acoustid chromaprint</PackageTags>
    <Version>1.3.3</Version>
    <AssemblyName>AcoustID</AssemblyName>
    <RootNamespace>AcoustID</RootNamespace>
    <Configurations>Debug;Release</Configurations>
    <PackageLicenseUrl>https://github.com/acoustid/chromaprint/blob/master/LICENSE.md</PackageLicenseUrl>
  </PropertyGroup>
```

Reply:
Changed have been implemented after the revert. See `AcoustID.csproj` in 085088c